assets/scripts/config/regions.config.js

```diff
@@ -190,6 +190,14 @@ export default {
       ap1: 'ap-northeast-1',
       gov: 'N/A'
     },
+    aws_private_link_cross_region: {
+      us: 'US East (N.Virginia) (us-east-1)',
+      us3: 'N/A',
+      us5: 'N/A',
+      eu: 'N/A',
+      ap1: 'TODO',
```

Comment:
@lowell-abraham-ddog Do you know what this one should be? It's referenced on line 269 in the README below